pytensor/link/numba/dispatch/basic.py

```diff
@@ -432,6 +433,46 @@ def shape_i(x):
     return shape_i
 
 
+@numba_funcify.register(SortOp)
+def numba_funcify_SortOp(op, node, **kwargs):
+    if op.kind == "quicksort":
+
+        @numba_njit
+        def sort_f(a, axis):
+            return np.sort(a)  # numba supports sort without arguments
+    else:
+        ret_sig = get_numba_type(node.outputs[0].type)
+
+        def sort_f(a, axis):
+            with numba.objmode(ret=ret_sig):
```

Comment:
instead of objmode just run in quicksort with a warning that it is ignored?

Same for argsort when the method is not provided.